src/components/AddUserForm/AddUserForm.js

```diff
@@ -0,0 +1,147 @@
+        //  было бы хорошо не пугать его висящим сообщением, что все таки что-то не так
+        this.setState({
+            currentError: '',
+            // errorUserName: '',
+            // errorUserLastName: ''
+            });
+        if (this.userNameRef.current.value !=='') {this.setState({errorUserName: ''})}
+        if (this.userLastNameRef.current.value !=='') {this.setState({errorUserLastName: ''})}
+
+    };
+
+    onSubmit = (e) => {
+        e.preventDefault();
+        if (this.userNameRef.current.value ==='' || this.userLastNameRef.current.value ==='') {
+            this.setState({
+                currentError: "Please input information",
+                errorUserName: this.userNameRef.current.value ==='' ? "INPUT NAME" : '',
+                errorUserLastName: this.userLastNameRef.current.value ==='' ? "INPUT LASTNAME" : ''
+            })
+        } else {
+        // Todo: добавить здесь проверку, если пользователь НЕ ВВЕЛ в инпуты ничего - не создаем пост
+        //  и обновляем currentError в стейте, кладем сообщение, что пошло не так
+        //  если пользователь ввел данные - создаем постзадачу
+
+        const {addUser} = this.props;
+
+        const user = {
+            name: this.userNameRef.current.value,
+            lastName: this.userLastNameRef.current.value,
+            id: uniqId()
+        };
+
+        addUser(user);
+
+        this.userNameRef.current.value = "";
+        this.userLastNameRef.current.value = "";
+        }
+    };
+
+    onReset = () => {
+        this.userNameRef.current.value = "";
+        this.userLastNameRef.current.value = "";
+        // todo: имплементнуть функцию скидывания значений, введеных пользователем
+    };
+
+    // onAlternativeSubmit = (event) => {
+    //     event.preventDefault();
+    //     const {addUser} = this.props;
+    //
+    //     const formData = new FormData(event.target);
+    //     debugger
+    //     const user = {
+    //         name: formData.get('userName'),
+    //         lastName: formData.get('userLastName'),
+    //         id: uniqId()
+    //     };
+    //     addUser(user);
+    //
+    //     console.log(user)
+    // };
+
+    render() {
+        return (
+            <form className={CN} onSubmit={this.onSubmit}>
+                <h2>Create new User</h2>
+                <div className="form-group">
+                    <label htmlFor="title" className="input-group-text">Enter post title:</label>
+                    <input
+                        ref={this.userNameRef}
+                        className={`form-control ${this.state.errorUserName  ? "error-input" : ""}`}
+                        type="text"
+                        name="userName"
+                        onChange={this.onInputChange}
+                    />
+                <div className={'error'}> {this.userNameRef.current ? this.state.errorUserName : ""}</div>
```

Comment:
Why do you have this condition `this.userNameRef.current ?` here ?

Reply:
I guess you can make a check in some other way

``
{ !!this.state.errorUserName  && <div className={'error'}> {this.state.errorUserName}</div> }
``